sdk/wallet-adapter/site/pages/advanced/wallet-standard.mdx

```diff
@@ -0,0 +1,147 @@
+# Wallet Standard
+
+Browser extension wallets built for Sui use the [Wallet Standard](https://github.com/wallet-standard/wallet-standard/) to define. This is a cross-chain standard that defines how wallets can automatically be discovered and interacted with from dapps.
+
+If you are building a wallet, we publish a helper library `@mysten/wallet-standard` which provides types and utilities that make it simple to get started.
+
```

Comment:
Side note: an "examples" sidebar item linking to sample implementations would be nice to have eventually

Reply:
Yeah going to punt this for now, but I agree.